t/ui/23-audit-log.t

```diff
@@ -130,9 +128,8 @@ subtest 'clickable events' => sub {
     wait_for_data_table;
     $table = $driver->find_element_by_id('audit_log_table');
     @entries = $driver->find_child_elements($table, 'tbody/tr', 'xpath');
-    is(scalar @entries, 2, 'three elements') or return diag $_->get_text for @entries;
+    is(scalar @entries, 1, 'three elements') or return diag $_->get_text for @entries;
```

Comment:
I know you didn't introduce the mistakes, but

* `three` is wrong here
* the `return` will result in exactly one `diag` call and will discard the rest of the `@entries`

Reply:
On, right :)

Reply:
done